javascript/index.js

```diff
@@ -39,27 +41,60 @@ function clearSplits() {
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  var start= document.getElementById("btnLeft")
+  start.innerHTML= 'Stop'
+  start.className.add("stop");
+ start.className.remove("star")
 }
 
 function setSplitBtn() {
+  var start= document.getElementById("btnRight");
+  start.innerHTML= 'Split'
+  start.className.add("split");
+  start.className.remove("reset")
+
+
   // ... your code goes here
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  var start= document.getElementById("btnLeft")
+  start.innerHTML= 'Start'
+  start.className.add("start");
+ start.className.remove("stop")
+ 
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  var start= document.getElementById("btnRight");
+  start.innerHTML= 'Reset'
+  start.className.add("reset");
+  start.className.remove("split")
+  
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
-  // ... your code goes here
+  chronometer.startClick()
```

Comment:
You are starting the chronometer twice. One time in line 78 and a second time in line 82. This will definitely be an issue running the code.